ci.sh

```diff
@@ -85,7 +85,7 @@ case $HOST_TARGET in
     MIRI_TEST_TARGET=i686-pc-windows-msvc run_tests
     MIRI_TEST_TARGET=x86_64-unknown-freebsd run_tests_minimal hello integer vec panic/panic concurrency/simple atomic data_race env/var
     MIRI_TEST_TARGET=aarch64-linux-android run_tests_minimal hello integer vec panic/panic
-    MIRI_TEST_TARGET=wasm32-wasi run_tests
+    MIRI_TEST_TARGET=wasm32-wasi MIRI_NO_STD=1 run_tests_minimal # supports std but miri doesn't support it
```

Comment:
`run_tests_minimal` expects as argument a filter to indicate which tests should be run. No filter means all tests.

However, `run_tests_minimal` also does not test `cargo miri`, so... it's not really helpful for this PR.